build/do.rq

```diff
@@ -187,20 +189,102 @@ fetch_engine_caps {
 }
 
 fetch_eopa_caps {
-	templ := {
-		"remote": "https://github.com/StyraInc/enterprise-opa.git",
-		"ref": "main",
-		"srcdir": "capabilities",
-		"destdir": eopa_caps_dir
+
+	# git ls-remote --tags output looks like this:
+	#
+	#	...
+	#	bab5bdd4bb2de45c7d1bcf5d7b5df3c61a131f38        refs/tags/v1.0.0
+	#	bab5bdd4bb2de45c7d1bcf5d7b5df3c61a131f38        refs/tags/v1.0.1
+	#	9ace37dda9859a1afd2fcd6a87dd7f1955be4dc4        refs/tags/v1.10.1
+	#	...
+	#
+	# so .stdout will look like this:
+	#
+	#	...
+	#	[
+	#		"bab5bdd4bb2de45c7d1bcf5d7b5df3c61a131f38\trefs",
+	#		"tags",
+	#		"v1.0.0"
+	#	],
+	#	[
+	#		"bab5bdd4bb2de45c7d1bcf5d7b5df3c61a131f38\trefs",
+	#		"tags",
+	#		"v1.0.1"
+	#	],
+	#	[
+	#		"bab5bdd4bb2de45c7d1bcf5d7b5df3c61a131f38\trefs",
+	#		"tags",
+	#		"v1.1.0"
+	#	],
+	#	...
+
+	print("fetching tags for enterprise-opa repository")
+
+	eopa_tags_result := rq.run([
+			"git",
+			"ls-remote",
+			"--tags",
+			"https://github.com/styrainc/enterprise-opa"
+		], {
+			"stdout_spec": {
+				"format": "raw",
+				"options": {
+					"raw.fs": "/",
+					"raw.rs": "[\n\r]"
+				},
+			}
+		}
+	)
+
+	{ rq.error(sprintf("\nstdout: %s\nstderr: %s\n", [eopa_tags_result.stdout, eopa_tags_result.stderr])) | eopa_tags_result.exitcode != 0 }
+
+	eopa_tags := {r[2] | r := eopa_tags_result.stdout[_]}
```

Comment:
Can we use `some ... in` iteration instead? Here and on line 257. This file isn't lintable at this point in time, as we'd need something similar for `rq` as this PR provides for `eopa`. But that doesn't mean should ignore the best practices and idioms Regal recommends.

Reply:
I'm not actually sure how to use `some ... in`, can that handle additional clauses? I ended up having to filter the tag list down with further clauses.

Reply:
hmm, yeah after 646a37db8db27a6f6661f280c37962cd8a806a99 I'm not sure `some ... in` can work, based on reading the docs

Reply:
It works anywhere:

```
eopa_tags := {t | some r in eopa_tags_result.stdout; t := r[2] ; not known_bad_tags[t]}
```

Reply:
Ahh I see, I didn't realize you could use it in that way. I'll switch it over.